### PR TITLE
Fixed potentially confusing error message after it led to misunderstanding with a user

### PR DESCRIPTION
**Pull request by agemooij**
The user got an error during blueprint validation saying "Streamlet port verbs not found for application". They had no idea what "streamlet port verbs were" so they asked support. Turns out it was an outlet called "verbs" on their streamlet that they had forgotten to declare but the lack of quotes made the sentence very hard to parse, even for me as the support engineer.

So I added some quotes.

Tested by running `sbt clean test` on the core.

### CODE REVIEW

File: core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala [hunk 1]
```diff
@@ -89,11 +89,11 @@ trait StreamletContext {
...
   case class MountedPathUnavailableException(volumeMount: VolumeMount)
-      extends Exception(s"Mount path for Volume Mount named [${volumeMount.name}] is unavailable.")
+      extends Exception(s"Mount path for Volume Mount named '${volumeMount.name}' is unavailable.")
 
 }
 
 case class PortNotFoundException(port: StreamletPort, streamletDefinition: StreamletDefinition)
     extends Exception(
-      s"Streamlet port ${port.name} not found for ${streamletDefinition.appId} and streamlet ${streamletDefinition.streamletRef}"
+      s"Streamlet port '${port.name}'' not found for application '${streamletDefinition.appId}' and streamlet '${streamletDefinition.streamletRef}'"
```
Comment (skyluc) [inline core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala]: Looks like you added too many quotes :)
```suggestion
      s"Streamlet port '${port.name}' not found for application '${streamletDefinition.appId}' and streamlet '${streamletDefinition.streamletRef}'"
```

Comment (agemooij) [reply core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala]: thanks @skyluc! My editor was clearly trying to be "smart" 😢

File: core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala [hunk 2]
```diff
@@ -89,11 +89,11 @@ trait StreamletContext {
       .getOrElse(throw MountedPathUnavailableException(volumeMount))
 
   case class MountedPathUnavailableException(volumeMount: VolumeMount)
-      extends Exception(s"Mount path for Volume Mount named [${volumeMount.name}] is unavailable.")
+      extends Exception(s"Mount path for Volume Mount named '${volumeMount.name}' is unavailable.")
```
Comment (RayRoestenburg) [inline core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala]:
```suggestion
      extends Exception(s"Mount path for volume mount named '${volumeMount.name}' is unavailable.")
```

File: core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala [hunk 3]
```diff
@@ -89,11 +89,11 @@ trait StreamletContext {
...
 
 }
 
 case class PortNotFoundException(port: StreamletPort, streamletDefinition: StreamletDefinition)
     extends Exception(
-      s"Streamlet port ${port.name} not found for ${streamletDefinition.appId} and streamlet ${streamletDefinition.streamletRef}"
+      s"Streamlet port '${port.name}' not found for application '${streamletDefinition.appId}' and streamlet '${streamletDefinition.streamletRef}'"
```
Comment (RayRoestenburg) [inline core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala]: Maybe it makes sense to add a `isOutlet` to `StreamletPort` (abstract), implement in `Inlet` and `Outlet`, or even add a `portType` to `StreamletPort` (abstract) and implement as `"inlet"` in `Inlet` and `"outlet"` in `Outlet` so we can be more precise in feedback to user..

Comment (agemooij) [reply core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala]: I agree, since the word "port" is not common for end-users and will get even more complex when we start adding gRPC and unmanaged connections. the more clear we can be in these errors the better is it.

That said, I'll merge this PR since I don't want to block it.
